versions.tf

```diff
@@ -1,10 +1,14 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.6"
 
   required_providers {
-    random = {
-      source  = "hashicorp/random"
-      version = ">= 3.0"
+    spacelift = {
+      source  = "spacelift-io/spacelift"
+      version = ">= 1.14"
+    }
+    external = {
+      source  = "hashicorp/external"
+      version = ">= 2.0"
```

Comment:
Doesn't look like we're using external any longer?

Reply:
Thanks, removed.